### DISCUSSION TITLE
提交图书馆版本

### DISCUSSION THREAD

**Discussion by Yongsgithub** [sikouhjw/gdutthesis#25] [Q&A] [answered]
最后上传图书馆收藏版本是PDF和WORD版本都要吗？今年会不会有改动？

**Comment by sikouhjw** [comment, marked as answer]
> 最后上传图书馆收藏版本是PDF和WORD版本都要吗？

研究生院某科长说中间不需要交 Word 之类的文件，也就是说中间和最终都只需要 pdf 格式的文件。

某些学院可能需要交一些 Word 做的表，这边准备的方案是用 LaTeX 仿制一份。

> 今年会不会有改动？

应该是不会有改动的，如果真的需要交 Word，就用 Word 模板[^1]吧。

[^1]: https://github.com/sikouhjw/gdutthesis-Word
